scripts/core/INTRO.rst

```diff
@@ -404,6 +404,25 @@ Specific environment variables can be set to control the behavior of unified run
 
     This environment variable is default enabled on Linux, but default disabled on Windows.
 
+CTS Environment Variables
+-------------------------
+
+The following environment variables are used by the CTS runner and can be used to specify the platform that the test
+framework should run on. This can be used during development and testing to run CTS tests in case multiple platforms
+are available. If both filters are specified, then they both must match a platform for it to be selected. If there are
+no valid platforms, then the tests will fail. Command line arguments take priority over these variables.
+
+.. envvar:: UR_CTS_ADAPTER_PLATFORM
+
+    A specifier list in the form of `[(backend):](platform name)[;[(backend)]:(platform name)]...`. For example:
+    `Fictional Corp(R);LEVEL_ZERO:Another Fictional Corp(R)`. If a backend specific specifier is present in the list and
```

Comment:
This seems like a confusing schema to me. How come the backend is optional?

Reply:
To preserve the original behaviour and provide a convenience for users who only want to run the tests on one device. I've expanded the documentation to hopefully explain things better.